README.md

```diff
@@ -77,7 +76,7 @@ const rpc = new JsonRpc('http://127.0.0.1:8888', { fetch });
 
 ### API
 
-Include textDecoder and textEncoder when using in Node, React Native, IE11 or Edge Browsers.
+Include textDecoder and textEncoder when using in Node. You may exclude these when running in a browser.
```

Comment:
IE11 still needs this right?

Reply:
Yep -- https://caniuse.com/#feat=textencoder. Updated the README with more instructions.